### ISSUE TITLE
Installer is not working correctly

### ISSUE THREAD

**Issue by jinxthomas** [tomaustin700/YAML-Pipeline-Validator-VS-Code-Extension#4]
**Describe the bug**
After installing the extension (which happens instantaneously) running the shortcut
 command 'azure-devops-pipeline-validator.validate' not found

When you try to access the command from Ctl-Shift-P the command is not listed

**To Reproduce**
Uninstall program from VSCode (if it already exists)
Close and restart VSCode
Open VSCode and go to extensions
Install YAML-Pipeline-Validator-VS-Code-Extension
Try to validate a YAML file
observe that the command does not exist

**Expected behavior**
I would expect the command to exist or that I would get an error when trying to install.

**Screenshots**
![image](https://user-images.githubusercontent.com/17104013/95083967-d7ede600-06e2-11eb-880a-2517d082a932.png)

**Comment by micheleissa** [comment]
@tomaustin700  I second this after the latest version I get the same error 
![image](https://user-images.githubusercontent.com/16201881/95139664-a43aac80-0732-11eb-8666-20b08adbeb66.png)

**Comment by tomaustin700** [comment]
Hi @micheleissa @jinxthomas 
Looks like something wasn't packaged correctly when I pushed 1.5.0 so it was missing some packages causing the extension to not load correctly. I have just pushed 1.5.2 which I think resolves the issue. Sorry about that and thanks for letting me know.

**Comment by jinxthomas** [comment]
Thanks so much for jumping right on this.  I've already shared your extension with two colleagues :)
    On Tuesday, October 6, 2020, 02:26:58 AM CDT, Tom Austin <notifications@github.com> wrote:  
 
 


Hi @micheleissa @jinxthomas
Looks like something wasn't packaged correctly when I pushed 1.5.0 so it was missing some packages causing the extension to not load correctly. I have just pushed 1.5.2 which I think resolves the issue. Sorry about that and thanks for letting me know.
